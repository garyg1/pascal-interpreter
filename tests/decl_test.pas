program decl_test;


var a : double = 1;
var b : boolean = false;

var c : double = 1;
    c1, c2 : boolean;
    d : double = 2;
    e : boolean = true;

const f : double = 1;
    g : boolean = false;
    h = true;

begin
    Writeln(a);
    Writeln(b);
    Writeln(c);
    Writeln(c1, '|', c2); // should get default value
    Writeln(d);
    Writeln(e);
    Writeln(f);
    Writeln(g);
    Writeln(h);
end.