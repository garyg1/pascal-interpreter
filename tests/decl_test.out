1.0
false
1.0
false|false
2.0
true
1.0
false
true
